### DISCUSSION TITLE
想问大家一个问题，在做后台管理系统时，下拉框，会先显示id，等接口数据拿到后，匹配中了，然后显示name；中间可能会出现较长的时间显示id值，想问下大家是怎么处理的？还是

### DISCUSSION THREAD

**Discussion by Cyxwmm** [ant-design/ant-design#32637] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
用户体验

### What does the proposed API look like?
可不可以在没匹配到值得情况下，显示空？

**Comment by yoyo837** [comment]
数据没加载好 你可以给组件设置为loading, 没必要非在项目上下功夫

**Comment by AlanCutFlim** [comment]
labelInValue

**Comment by vVvlife** [comment]
异步预加载
